CHANGELOG

```diff
@@ -0,0 +1,30 @@
+# Changelog Since v0.6.0
+
+## New Features
+
+- Adding `PodSecurityPoliciy` to allow `csi-gce-pd-node` in clusters with policies enabled.
```

Comment:
Should this be under some "action required" section to make it more clear that user action may be required? (Instead of all caps)

Reply:
yeah thats a good QOL thing